src/main/java/org/apache/maven/plugins/javadoc/JavadocUtil.java

```diff
@@ -221,14 +221,17 @@ protected static List<String> getExcludedPackages(
      * @return quoted option-argument
      */
     protected static String quotedArgument(String value) {
+        if (value == null) {
+            return null;
+        }
         String arg = value;
 
+        List<String> list = Arrays.stream(arg.split("\n")).map(String::trim).collect(Collectors.toList());
+        arg = String.join("", list);
```

Comment:
try to avoid reusing variables as it's bug prone

Also, this is combining lines with no space in between one line and the next. Is that what you want?

Reply:
> try to avoid reusing variables as it's bug prone
> 
yup please look at the context of the code (the full code) yes it's not idea but it;s how the rest of the code is and I don't want to change everything this could be part of another PR if you like.

> Also, this is combining lines with no space in between one line and the next. Is that what you want?